### PR TITLE
Adding Gov docs and updating community guides

### PR DESCRIPTION
**Pull request by blackbera**
## Description

What changes are made in this PR? Is it a feature or a bug fix?
Adding Gov docs and updating community guides
Does it close a specific issue?

Example:

```
Closes #1
```

## Contribution

- [ ] I have followed the [Development Workflow](https://github.com/berachain/docs/blob/main/CONTRIBUTING.md#development-workflow)
- [ ] I have read the [CODE OF CONDUCT](https://github.com/berachain/docs/blob/main/CODE_OF_CONDUCT.md)

Let us know your wallet address/ENS:

```
0xYourAddress
```

### CODE REVIEW

File: apps/core/content/learn/governance/gauges.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   - Proposals are submitted on-chain
+   - 3-hour waiting period before voting begins
+
+3. **Voting Period**:
+   - 3-hour active voting window
+   - BGT holders cast votes (quorum: 2B BGT)
+
+4. **Proposal Outcome**:
+   - If passed: Enters a 2-day timelock queue
+   - If declined: Marked as defeated, allowing for a new proposal after addressing concerns
```
Comment (corduroybera) [inline apps/core/content/learn/governance/gauges.md]: These are testnet placeholders, they may or may not change before governance is live

File: apps/core/content/learn/governance/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+3. **Prepare Your Proposal**: Clearly define the changes you want to implement. For example, to add a Reward Vault to the BeraChef contract, you would need to specify the vault address and the function call to update the "friends of the chef."
+
+4. **Submit the Proposal**: Use the governance contract to submit your proposal on-chain. This typically involves calling a `propose` function with the necessary parameters.
+
+5. **Monitor and Participate**: Once submitted, monitor your proposal's progress through the various stages. Encourage other $BGT holders to vote and participate in discussions about your proposal.
+
+6. **Execute if Passed**: If your proposal passes and completes the timelock period, you or another user can execute it to implement the changes.
+
+For a detailed walkthrough of creating a governance proposal, including code examples and step-by-step instructions, please refer to our [full tutorial](https://github.com/berachain/rewards-vault-tutorial).
```
Comment (corduroybera) [inline apps/core/content/learn/governance/index.md]: link 404s

File: apps/core/content/learn/governance/gauges.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Understanding Reward Vaults in PoL
+
+Reward Vaults are smart contracts that enable validators to share their BGT rewards with protocols. When validators build blocks, they earn BGT proportional to their staked amount. Validators can then:
+
+1. Use BGT for governance participation
+2. Exchange BGT for Bera (the native gas token)
+3. Share BGT rewards with protocols through Reward Vaults
+
+This mechanism is central to Berachain's [Reward Vaults](https://docs.berachain.com/developers/contracts/rewards-vault) system.
```
Comment (corduroybera) [inline apps/core/content/learn/governance/gauges.md]:
```suggestion
This mechanism is central to Berachain's [Reward Vaults](/developers/contracts/rewards-vault) system.
```

Switch to relative

File: apps/core/.vitepress/sidebar.ts [hunk 4]
```diff
@@ -47,6 +47,28 @@ const SIDEBAR = {
...
+        {
+          text: "Delegate BGT To Validator ",
+          link: "/learn/guides/delegate-bgt-to-validator",
+        },
+        {
+          text: "Create PoL Gauge Proposal  ",
+          link: "/learn/guides/create-pol-guage-proposal",
+        },
+        { text: "Create Incentive", link: "/learn/guides/create-incentive" },
+        { text: "Burn $BGT", link: "/learn/guides/burn-bgt" },
```
Comment (corduroybera) [inline apps/core/.vitepress/sidebar.ts]: Dead links showing in sidebar

File: apps/core/content/learn/governance/gauges.md
Comment (corduroybera) [inline apps/core/content/learn/governance/gauges.md:1]: This page should have some reference to how to create a reward vault. Maybe as a step 0 or pre-requisite, just to make sure it's clear how they get it deployed prior to proposal

**Review by corduroybera — CHANGES_REQUESTED** [review]

File: apps/core/.vitepress/sidebar.ts [hunk 5]
```diff
@@ -47,6 +47,28 @@ const SIDEBAR = {
...
+      items: [
+        { text: "Governance Overview", link: "/learn/governance/" },
+        { text: "Reward Vault Whitelisting", link: "/learn/governance/gauges" },
+      ],
+    },
+    {
+      text: "Guide",
+      items: [
+        {
+          text: "Delegate BGT To Validator ",
```
Comment (codingwithmanny) [inline apps/core/.vitepress/sidebar.ts]:
```suggestion
          text: "Delegate BGT To Validator",
```

File: apps/core/.vitepress/sidebar.ts [hunk 6]
```diff
@@ -47,6 +47,28 @@ const SIDEBAR = {
...
+          link: "/learn/guides/delegate-bgt-to-validator",
+        },
+        {
+          text: "Create PoL Gauge Proposal  ",
```
Comment (codingwithmanny) [inline apps/core/.vitepress/sidebar.ts]:
```suggestion
          text: "Create PoL Gauge Proposal",
```

File: apps/core/content/developers/guides/community-guides.md [hunk 7]
```diff
@@ -30,24 +40,42 @@ These are a list of community developer guides for Berachain.
...
 | ------------------------------------ | ------------------------------------------------------------------------- |
 | Using Gelato VRF                     | [GitHub](https://github.com/berachain/guides/tree/main/apps/gelato-vrf)   |
 | Provably Fair NFTs with Pyth Entropy | [GitHub](https://github.com/berachain/guides/tree/main/apps/pyth-entropy) |
+| Pyth Oracle                          | [GitHub](https://github.com/berachain/guides/tree/main/apps/pyth-oracle)  |
+
+## Governance
+
+| Project Name                  | Git Repository                                                                             |
+| ----------------------------- | ------------------------------------------------------------------------------------------ |
+| Berachain Governance Proposal | [GitHub](https://github.com/berachain/guides/tree/main/apps/berachain-governance-proposal) |
```
Comment (codingwithmanny) [inline apps/core/content/developers/guides/community-guides.md]:
```suggestion
| Berachain Governance Proposal For Reward Vaults | [GitHub](https://github.com/berachain/guides/tree/main/apps/berachain-governance-proposal) |
```

File: apps/core/content/.assets/governance-process.png
Comment (codingwithmanny) [inline apps/core/content/.assets/governance-process.png:1]: Put this under `apps/core/content/public` or remove. Seems like a duplicate.

File: apps/core/content/learn/governance/gauges.md
Comment (codingwithmanny) [inline apps/core/content/learn/governance/gauges.md:1]: Revise page to be `rewardvaults.md` and adjust sidebar.

File: apps/core/content/learn/governance/gauges.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+# Creating a Governance Proposal for Berachain Reward Vaults
+
+Berachain's Proof of Liquidity (PoL) consensus mechanism allows protocols to bootstrap their liquidity by receiving Berachain Governance Token (BGT) emissions from validators. This process is facilitated through Reward Vaults, which play a crucial role in the governance and incentive structure of the Berachain ecosystem.
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/gauges.md]: Any reference to tokens, make the format `$BGT` as an example.

File: apps/core/content/learn/governance/gauges.md
Comment (codingwithmanny) [reply apps/core/content/learn/governance/gauges.md:1]: Agree that there should be some links. The only issue is that there really isn't a UI version of things, but perhaps mention that if you'd like to do it programmatically.. see here..

File: apps/core/content/learn/governance/gauges.md (hunk 1 above)
Comment (codingwithmanny) [reply apps/core/content/learn/governance/gauges.md]: Agree would just add a section under the title with something like:

```md
### Governance Process for Whitelisting

:::warning
**NOTE:** The values mentioned are currently on Testnet which are placeholders subject to change and may not reflect the values used for Mainnet.
:::
```

File: apps/core/content/learn/governance/index.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+    - name: description
+      content: How Berachain's Governance Works
+  - - meta
+    - property: og:description
+      content: How Berachain's Governance Works
+---
+
+# Berachain Governance
+
+Berachain's Governance system uses BGT to allow token holders to make important decisions about core functions of Proof of Liquidity & our core dapps. Here's a few examples of what governance can be used for:
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/index.md]:
```suggestion
Berachain's Governance system uses `$BGT` to allow token holders to make important decisions about core functions of Proof-of-Liquidity & our core dapps. Here are a few examples of what governance can be used for:
```

File: apps/core/content/learn/governance/index.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+- PoL asset whitelisting (e.g., new staking assets, whitelisting incentive assets)
+- $HONEY minting parameters (e.g., eligible collateral assets, minting rates)
+- Native dApp governance (e.g., changes to Bend's interest rate model)
+  - Berps
+  - Bend
+  - Bex
+- Whitelisting Rewards Vaults for BGT emissions
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/index.md]: Isn't the first related to the last item?

```suggestion
1. `$HONEY` minting parameters (e.g., eligible collateral assets, minting rates)
2. Native dApp governance for BEX, Berps, & Bend (e.g., changes to Bend's interest rate model)
3. Whitelisting Rewards Vaults for `$BGT` emissions
```

File: apps/core/content/learn/governance/index.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+
+
+## Governance Process
+
+The governance process on Berachain follows several stages:
+
+1. **Proposal Creation**: Any user with sufficient voting power (at least 1000 $BGT) can create a governance proposal.
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/index.md]:
```suggestion
1. **Proposal Creation**: Any user with sufficient voting power (meeting a minimum amount of `$BGT`) can create a governance proposal.
```

File: apps/core/content/learn/governance/index.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+- $HONEY minting parameters (e.g., eligible collateral assets, minting rates)
+- Native dApp governance (e.g., changes to Bend's interest rate model)
+  - Berps
+  - Bend
+  - Bex
+- Whitelisting Rewards Vaults for BGT emissions
+
+
+
+## Governance Process
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/index.md:26]: This section I would remove specifics on amounts of BGT and number of hours and add a section just after the steps with Testnet values.

Example.

```md
## Governance Process

The governance process on Berachain follows several stages.

![Governance Process](/assets/governance-process.png)

1. **Proposal Creation**: Any user with sufficient voting power (meeting a minimum amount of `$BGT`) can create a governance proposal.

...

### Testnet Governance Values

| State | Criteria |
| ----------- | ----------- |
| Proposal Creation | 1000 `$BGT` Required |
| Pending State | 3-hour waiting period |
| Proposal Outcome | 2 billion `$BGT` required reach quorum  |
...
```

File: apps/core/content/learn/governance/index.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+6. **Execution**: After the timelock period, the proposal can be executed, implementing the proposed changes.
+
+![Governance Process](/assets/governance-process.png)
+
+## Creating a Governance Proposal
+
+To create a governance proposal on Berachain:
+
+1. **Ensure Sufficient Voting Power**: You need at least 1000 $BGT, either owned directly or delegated to you by other token holders.
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/index.md]: See above comment and see where you can consolidate specific values to one section, aka remove 1000 $BGT.

File: apps/core/.vitepress/sidebar.ts (hunk 5 above)
Comment (blackbera) [reply apps/core/.vitepress/sidebar.ts]: airdrop farmer comment

File: apps/core/.vitepress/sidebar.ts (hunk 6 above)
Comment (blackbera) [reply apps/core/.vitepress/sidebar.ts]: airdrop farmer comment

File: apps/core/content/learn/governance/rewardvault.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+   - 3-hour waiting period before voting begins
+
+3. **Voting Period**:
+
+   - 3-hour active voting window
+   - `$BGT` holders cast votes (quorum: 2B `$BGT`)
+
+4. **Proposal Outcome**:
+   - If passed: Enters a 2-day timelock queue
+   - If declined: Marked as defeated, allowing for a new proposal after addressing concerns
```
Comment (codingwithmanny) [inline apps/core/content/learn/governance/rewardvault.md]: Review this section to remove the numbers (3-hour, 2-day, 1000 BGT) and refer to other page for values.

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Review by codingwithmanny — CHANGES_REQUESTED** [review]
Add versioning

**Pull request merged**